core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -250,14 +250,14 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
       assertResult(
         Some(
           ExecutionServiceWorkflowOptions(
-            jes_gcs_root = s"gs://${testData.workspace.bucketName}/${testData.submission1.submissionId}",
+            jes_gcs_root = s"gs://${testData.workspace.bucketName}/submissions/${testData.submission1.submissionId}",
```

Comment:
This looks like to me like a change in default behavior. Where is this extra `submissions/` path coming from?

Reply:
This isn't a behavioral change, I just changed the test data to flex it a bit: https://github.com/broadinstitute/rawls/pull/1896/files#diff-e6dd2e193c5a6757c3575da466015485f9ff152b74db02ffc76a0b5f21bcdf10R154